workflows/elevenlabs.io/workflows.arazzo.json

```diff
@@ -217,55 +110,53 @@
       ]
     },
     {
-      "workflowId": "clone-voice-from-samples-and-generate-speech",
-      "summary": "Uploads audio samples to create a new cloned voice, then uses that voice to generate speech from text.",
-      "description": "Uploads audio samples to create a new cloned voice, then uses that voice to generate speech from text.",
+      "workflowId": "conversational-ai-agent-creation-and-configuration",
+      "summary": "Creates a conversational AI agent, configures its knowledge base, and retrieves its widget configuration. This workflow allows users to set up and customize an AI agent for conversational interactions.",
+      "description": "Creates a conversational AI agent, configures its knowledge base, and retrieves its widget configuration. This workflow allows users to set up and customize an AI agent for conversational interactions.",
       "inputs": {
         "type": "object",
         "properties": {
-          "xi-api-key": {
-            "type": "string"
-          },
-          "name": {
-            "type": "string"
-          },
-          "files": {
+          "use_tool_ids": {
             "type": "string"
           },
-          "voice_id": {
+          "xi-api-key": {
             "type": "string"
           },
-          "enable_logging": {
+          "conversation_config": {
             "type": "string"
           },
-          "optimize_streaming_latency": {
-            "type": "string"
+          "add-to-knowledge-base_body": {
+            "type": "object"
           },
-          "output_format": {
+          "agent_id": {
```

Comment:
No need for `agent_id` input. It's going to be generated by `create-agent` step.